app/routes.py

```diff
@@ -129,6 +129,7 @@ def index():
                            logo=render_template(
                                'logo.html',
                                dark=g.user_config.dark),
+                           config_disabled=os.getenv('WHOOGLE_CONFIG_DISABLE_CHANGE', ''),
```

Comment:
I'd prefer if this was an app level config setting (i.e. in `app/__init__.py`). Then access it via `app.config['CONFIG_DISABLED']` or whatever, rather than reading from the environment variable directly.